src/types/onyx/Policy.ts

```diff
@@ -141,12 +141,12 @@ type QBOConnectionData = {
     vendors: Vendor[];
 };
 
-type IntegrationEntityMap = 'NONE' | 'DEFAULT' | 'TAG' | 'REPORT_FIELD';
+type IntegrationEntityMap = (typeof CONST.INTEGRATION_ENTITY_MAP_TYPES)[keyof typeof CONST.INTEGRATION_ENTITY_MAP_TYPES] | false | true;
```

Comment:
```suggestion
type IntegrationEntityMap = (typeof CONST.INTEGRATION_ENTITY_MAP_TYPES)[keyof typeof CONST.INTEGRATION_ENTITY_MAP_TYPES];
```

Can we get rid of `true` and `false` if we have `IMPORTED` and `NOT_IMPORTED`?

Reply:
As i mentioned above - for now we could not move away from true/false - it should be done on API level - and should not be a blocker for current PR - probably that change will be done as a follow up @eVoloshchak